src/nft/nft.controller.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  Body,
+  Controller,
+  Get,
+  NotFoundException,
+  Param,
+  Post,
+  Res,
+  ServiceUnavailableException,
+  UploadedFile,
+  UseInterceptors,
+} from '@nestjs/common';
+import { FileInterceptor } from '@nestjs/platform-express';
+import { Response } from 'express';
+import { StorageFile } from 'src/configs/google-storage/google-storage';
+import { StorageService } from 'src/storage/storage.service';
+
+@Controller('nft')
+export class NftController {
+  constructor(private storageService: StorageService) {}
+
+  @Post()
+  @UseInterceptors(
+    FileInterceptor('file', {
+      limits: {
+        files: 1,
+        fileSize: 1024 * 1024,
+      },
+    }),
+  )
+  async uploadMedia(
+    @UploadedFile() file: Express.Multer.File,
+    @Body('mediaId') mediaId: string,
```

Comment:
I don't understand what is `mediaId`. Is that a id of the previously created Nft?